osbuild-composer.spec

```diff
@@ -117,6 +118,7 @@ export GOFLAGS=-mod=vendor
 %gobuild -o _bin/osbuild-composer %{goipath}/cmd/osbuild-composer
 %gobuild -o _bin/osbuild-worker %{goipath}/cmd/osbuild-worker
 
+make man
```

Comment:
Since this is the first time we use make in our spec file, I'd also like to have an explicit "make" buildrequires for it. It is currently included in all buildroots by default, but it will bite us in few months:
https://fedoraproject.org/wiki/Releases/34/ChangeSet#Remove_make_from_BuildRoot

Reply:
I added make explicitly as BuildRequires. Thanks for pointing out the Fedora change.